### PR TITLE
fix: Upgrade Markdownlint-CLI2 and address issues

### PR DESCRIPTION
**Pull request by nschonni**
### Description

Update the new version and fix the new issues

### Motivation

The auto-update issue https://github.com/mdn/translated-content/pull/10863 is failing because of the new flagged issues. The auto-fix for the issues also appears to be incorrect, so had to manually address the issues

### CODE REVIEW

**Comment by nschonni** [comment]
@DavidAnson you may want to add this repo to the CI tests too. The issues that it was flagging were correct, but the autofixes were actually bad. For the weird `https://` inside an absolute link, it was trying to treat them as a bare URL

File: files/zh-tw/learn/html/multimedia_and_embedding/images_in_html/index.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ original_slug: Learn/HTML/Multimedia_and_embedding/HTML中的圖片
 
 > **備註：** \<img>和\<video>之類的元素有時也稱為替換元素。 這是因為元素的內容和圖片大小是由外部（例如圖片或影音檔）所定義的，而不是由元素的內容定義。
 
-> **備註：** 您可以從在[Github](/zh-TW/docs/https://mdn.github.io/learning-area/html/multimedia-and-embedding/images-in-html/index.html)上找到本節完成的示例（參見[開源碼](/zh-TW/docs/https://github.com/mdn/learning-area/blob/master/html/multimedia-and-embedding/images-in-html/index.html)。）
+> **備註：** 您可以從在[Github](/zh-TW/docs/<https://mdn.github.io/learning-area/html/multimedia-and-embedding/images-in-html/index.html)上找到本節完成的示例（參見[開源碼>](https://github.com/mdn/learning-area/blob/master/html/multimedia-and-embedding/images-in-html/index.html)。）
```
Comment (yin1999) [inline files/zh-tw/learn/html/multimedia_and_embedding/images_in_html/index.md:76]:
```suggestion
> **備註：** 您可以從在 [Github](https://mdn.github.io/learning-area/html/multimedia-and-embedding/images-in-html/index.html) 上找到本節完成的示例（參見[開源碼](https://github.com/mdn/learning-area/blob/master/html/multimedia-and-embedding/images-in-html/index.html)。）
```

**Review by yin1999 — APPROVED** [review]
LGTM for l10n-zh

**Comment by DavidAnson** [comment]
@nschonni I will have a look at adding this to the list of test repositories, thank you!

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja`.

**Review by SphinxKnight — APPROVED** [review]
approved for @mdn/yari-content-fr Thanks once more @nschonni 🙇

**Review by sashasushko — APPROVED** [review]
Approve from Ru. Thanks 🙏

**Review by Graywolf9 — APPROVED** [review]
LGTM for `es`, thank you!

**Review by hochan222 — APPROVED** [review]
LGTM for `ko`, thank you!

**Comment by hochan222** [comment]
Probably only `pt-br` locale remains.

**Comment by nschonni** [comment]
The one pt-br item is removing a `title` from a link. I think it should be OK to land

/cc @caugner @bsmth

**Review by josielrocha — APPROVED** [review]
LGTM for @mdn/yari-content-pt-br.
Thank you @nschonni!

**Review by yin1999 — COMMENTED** [review]
The part of yarn package changes are approved by mdn bot in #10863. It's time to merge this.

**Comment by DavidAnson** [comment]
@nschonni, per your suggestion above, this repo is now part of the `markdownlint` CI tests: <https://github.com/DavidAnson/markdownlint/commit/ff85dde7dfbe0f2f893656a217bf4ec7018399ba>.

**others**, this means I'm less likely to introduce a linting regression for this repository.

**Pull request merged**